beacon-node-oapi.yaml

```diff
@@ -95,6 +95,8 @@ paths:
     $ref: "./apis/beacon/blocks/root.yaml"
   /eth/v1/beacon/blocks/{block_id}/attestations:
     $ref: "./apis/beacon/blocks/attestations.yaml"
+  /eth/v1/beacon/blobs_sidecars/{block_id}:
+    $ref: "./apis/beacon/blobs_sidecars/blobs_sidecar.yaml"
```

Comment:
do both need to be plurals? is `blob_sidecars` accurate?

if the paths / files / object names can be consistent that'll make it easier to navigate

Reply:
hmm I think this is on an old commit as well, I just did a global search for `blobs_` and not finding anything. I'll make sure the paths / file name are matching if we end up changing the paths 👍